Library/Homebrew/extend/os/linux/install.rb

```diff
@@ -51,5 +70,23 @@ def symlink_ld_so
       FileUtils.ln_sf ld_so, brew_ld_so
     end
     private_class_method :symlink_ld_so
+
+    def symlink_gcc_libs
+      gcc_optprefix = HOMEBREW_PREFIX/"opt"/OS::LINUX_PREFERRED_GCC_FORMULA
+
+      GCC_RUNTIME_LIBS.each do |library|
+        gcc_library = gcc_optprefix/"lib/gcc"/GCC_VERSION_SUFFIX/library
+        gcc_library_symlink = HOMEBREW_PREFIX/"lib"/library
+        # Skip if the link target doesn't exist.
+        next unless gcc_library.readable?
+
+        # Also skip if the symlink already exists.
+        next if gcc_library_symlink.readable?
```

Comment:
What if it exists but points to the wrong place?

Reply:
Good point - I believe the check needs to also make sure that the `realpath` equals `gcc_library`.  So maybe something like:
```ruby
if gcc_library_symlink.readable?
  next if gcc_library_symlink.realpath == gcc_library
end
```
Basically I'm trying to avoid checking `gcc_library_symlink.realpath` unless `gcc_library_symlink` exists. If there's a more concise way to express this I'm happy to change it.